src/test/ui/consts/rfc-2203-const-array-repeat-exprs/const-repeat.rs

```diff
@@ -0,0 +1,27 @@
+// check-pass
```

Comment:
`run-pass` to check that destructor does not get called

Reply:
Ah, good catch!